.github/workflows/release.yml

```diff
@@ -126,46 +126,22 @@ jobs:
           name: containerd-binaries-${{ matrix.os }}
           path: src/github.com/containerd/containerd/*.tar.gz*
 
-      - name: Install cri-containerd dependencies
+      - name: Make cri-containerd tar
         shell: bash
         env:
           RUNC_FLAVOR: runc
-          DESTDIR: ${{ github.workspace }}/cri-release
         run: |
-          mkdir ${DESTDIR}
           if [[ "${OS}" == "linux" ]]; then
-            sudo install -d ${DESTDIR}/usr/local/bin
-            sudo install -D -m 755 bin/* ${DESTDIR}/usr/local/bin
-            sudo install -d ${DESTDIR}/opt/containerd/cluster
-            sudo cp -r contrib/gce ${DESTDIR}/opt/containerd/cluster/
-            sudo install -d ${DESTDIR}/etc/systemd/system
-            sudo install -m 644 containerd.service ${DESTDIR}/etc/systemd/system
-            echo "CONTAINERD_VERSION: '${RELEASE_VER#v}'" | sudo tee ${DESTDIR}/opt/containerd/cluster/version
-
             sudo PATH=$PATH script/setup/install-seccomp
```

Comment:
any reason to leave just seccomp here?

Reply:
seccomp is an environment dependency for build rather than a package which gets built along with the release. It doesn't make sense to have the makefile install this since it may already be installed in the environment in some other way.